test/functions.test.js

```diff
@@ -40,53 +93,1521 @@ test('greetUsersOverAge60', (expect) => {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Mercury': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Mitsubishi': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Oldsmobile': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Peugeot': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Pontiac': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Porsche': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Scion': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Subaru': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Toyota': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        },
+        'Volvo': {
+            'Bigender': 1,
+            'Female': 19,
+            'Genderqueer': 1,
+            'Male': 23,
+            'Non-binary': 1
+        }
+    };
 
-    const actual = true;
+    const actual = getGenderBreakdownOfEachCar(customers);
 
-    expect.equal(actual, expected);
+    expect.deepEqual(actual, expected);
 });
 
-skip('getAllCoolFactorsOfEachCar', (expect) => {
-    const expected = true;
+test('getAllCoolFactorsOfEachCar', (expect) => {
+    const expected = {
+        'Alfa Romeo': [
+            4,
+            9,
+            4,
+            4,
+            6,
+            2,
+            9,
+            8,
+            2,
+            2,
+            4,
+            5,
+            1,
+            4,
+            6,
+            10,
+            3,
+            7,
+            8,
+            9,
+            3,
+            9,
+            6,
+            2,
+            5,
+            3,
+            1,
+            10,
+            8,
+            1,
+            8,
+            10,
+            2,
+            8,
+            2,
+            10,
+            5,
+            1,
+            1,
+            8,
+            5,
+            3,
+            2,
+            7,
+            2
+        ],
```

Comment:
really similar issue here to the gender breakdown by car type. this is getting all of the cool factors and putting them in each car type. Need to be able to find the cool factor for all alfa romeo owners, audi owners, etc. and it'll look more like `Alfa Romeo: [6], Audi: [1], Chevrolet: [8, 9, 2, 1], ...`